sysutils/bash-completion/files/patch-xine-add-more-formats.diff

```diff
@@ -4,27 +4,23 @@ Origin: vendor
 Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/bash-completion/+bug/311525
 Bug-Debian: https://bugs.debian.org/496162
```

Comment:
If this patch is no longer the version provided by these Debian/Ubuntu, we should delete/update the references. At least from the Debian source, I could not yet find an updated patch for 2.12.

https://salsa.debian.org/debian/bash-completion/-/blob/master/debian/patches/04-xine_add_more_formats.patch?ref_type=heads